src/main/java/com/gildedrose/AgedBrieUpdateStrategy.java

```diff
@@ -0,0 +1,25 @@
+package com.gildedrose;
+
+public class AgedBrieUpdateStrategy implements UpdateStrategy {
+
+    @Override
+    public boolean matches(Item item) {
+        return item.name.equals("Aged Brie");
+    }
+
+    @Override
+    public int getNewQuality(Item item) {
+        if (item.quality >= 50) {
```

Comment:
extract quality and sell in limits to (global) constants, reusable in all strategies.